package.json

```diff
@@ -26,33 +25,40 @@
   },
   "homepage": "https://github.com/resin-io-modules/resin-bundle-resolve#readme",
   "devDependencies": {
-    "@types/bluebird": "^3.0.37",
-    "@types/chai": "^3.4.35",
-    "@types/lodash": "^4.14.52",
-    "@types/mocha": "^2.2.39",
-    "@types/node": "^7.0.7",
-    "@types/request": "0.0.45",
-    "@types/semver": "^5.3.32",
-    "chai": "^3.5.0",
-    "husky": "^0.14.3",
-    "lint-staged": "^8.1.0",
-    "mocha": "^6.0.0",
-    "prettier": "^1.15.3",
+    "@types/bluebird": "^3.5.26",
+    "@types/chai": "^4.1.7",
+    "@types/lodash": "^4.14.122",
+    "@types/mocha": "^5.2.6",
+    "@types/node": "^8.10.45",
+    "@types/request": "^2.48.1",
+    "@types/semver": "^6.0.0",
+    "chai": "^4.2.0",
+    "husky": "^1.3.1",
+    "lint-staged": "^8.1.5",
+    "mocha": "^6.1.2",
+    "prettier": "^1.16.4",
     "require-npm4-to-publish": "^1.0.0",
     "resin-lint": "^2.0.1",
     "tar-utils": "^1.1.0",
-    "tslint": "^5.5.0",
-    "typescript": "^2.1.6"
+    "tslint": "^5.15.0",
+    "typescript": "^3.4.3"
   },
   "dependencies": {
+    "@types/chai-as-promised": "^7.1.0",
```

Comment:
`devDependencies` instead `dependencies`. Btw, am I right to understand that **all** `@types` dependencies are always `devDependencies`?

Reply:
Fixed, moved both types and actual module to dev dependencies.  For `@types` in general though they should match wherever the main module is, the reason being that downstream will need the types to be installed/available, and at the right version, in order use them for type-checking (our generated `.d.ts` files will likely refer to them)